tests/terraform/checks/resource/aws/test_S3BucketObjectLock.py

```diff
@@ -46,22 +44,35 @@ def test_success(self):
         scan_result = check.scan_resource_conf(conf=resource_conf)
         self.assertEqual(CheckResult.PASSED, scan_result)
 
-    def test_success_no_param(self):
+    def test_dynamic_value_object_lock(self):
```

Comment:
@mourlos Thanks!
Can we add another test, instead of replacing that test?

Reply:
Hi @nimrodkor , I had a look on the changes and looks like I am not handling the check correctly at this stage and the original check works fine (the issue was on my initial tf code that I was testing against).

Given the above, this PR doesn't really add any real value, apart from the extra test case, in which case I am not sure it's worth merging.

Feel free to close it in this case (or merge as part of an actual release)